bot/seasons/evergreen/nationday.py

```diff
@@ -0,0 +1,149 @@
+        """
+        # Get current date [Month and day]
+        date = datetime.datetime.today()
+        month = date.strftime("%B")
+        day = date.day
+        today_date = f'{month} {day}'
+        today_date = today_date.lower()
+        try:
+            # Get list of countries
+            countries = self.by_day[today_date]
+            countries = list(countries.split(','))
+        except Exception as ke:
+            err_msg = f"No country has an independence day today. {ke}"
+            logger.warning(err_msg)
+            return [(err_msg, white_flag)]
+        # Create pages
+        pages = []
+        for country in countries:
+            page, img = await self.get_country_info(country)
+            pages.append((page, img))
+        return pages
+
+    async def get_country_info(self, country: str) -> Tuple[str, str]:
+        """
+        Create country information page using RESTCountries API.
+
+        Return page and flag image.
+        """
+        try:
+            iso_code = self.iso_codes[country]
+            result = requests.get(url+iso_code)
+            info = result.json()
+
+            country_info = ""
+            country_info += f"Region: {info['region']}\n\n"
+            country_info += f"Capital: {info['capital']}\n\n"
+            country_info += f"Population: {info['population']}\n\n"
+            country_info += f"Currency: {info['currencies'][0]['name']} ({info['currencies'][0]['symbol']})\n"
+            flag = flag_url+iso_code+"/flat/64.png"
+
+        except Exception as e:
+            msg = f"Error! {e}"
+            logger.warning(e)
+            return msg, white_flag
+
+        return country_info, flag
+
+    @command(name='nationday')
+    async def nationday(self, ctx: Context, param: str = "") -> None:
+        """
+        \U0001F30F NationDay Help.
+
+        Enter a country name to get independence day of that country along with some basic information on the country.
+        Enter "today" to get all countries whose independence day is the current day, along with information.
+        Usage:
+        -> .nationday today
+        -> .nationday [country]
+        Examples:
+        -> .nationday "india"
+        -> .nationday "South Sudan"
+        -> .nationday today
+        """
+        param = param.lower()
+
+        if param == 'Today':
+            pages = await self.country_today()
+            embed = discord.Embed(
+                title='Countries that have their independence days today')\
+                .set_footer(text='Powered by the RESTCountries API.')
+            await ImagePaginator.paginate(pages, ctx, embed)
+
+        # Check if country is present
+        elif param in self.iso_codes.keys():
+            page, date = await self.get_specific_country(param)
+            if page[0][0] is not None:
+                embed = discord.Embed(
+                    title=f'{param} -> {date}')\
+                    .set_footer(text='Powered by the RESTCountries API.')
```

Comment:
Please try to avoid breaking lines with `\`, this can be written as follows:
```python
embed = discord.Embed(
    title=f'{param} -> {date}'
).set_footer(text='Powered by the RESTCountries API.')
```
Same case on lines 128, 129 above.

Reply:
Thanks for all the suggestions! Will look into it soon.